gridstatus/tests/test_ieso.py

```diff
@@ -0,0 +1,223 @@
+
+        # date string works
+        date_str = test_date.strftime("%Y%m%d")
+        df = self.iso.get_load(date_str)
+        self._check_load(df)
+        assert df.loc[0]["Interval Start"].strftime("%Y%m%d") == date_str
+
+        # timestamp object works
+        df = self.iso.get_load(test_date)
+
+        self._check_load(df)
+        assert df.loc[0]["Interval Start"].strftime(
+            "%Y%m%d",
+        ) == test_date.strftime("%Y%m%d")
+
+        # datetime object works
+        df = self.iso.get_load(test_date)
+        self._check_load(df)
+        assert df.loc[0]["Interval Start"].strftime(
+            "%Y%m%d",
+        ) == test_date.strftime("%Y%m%d")
+
+    def test_get_load_latest(self):
+        df = self.iso.get_load("latest")
+        self._check_load(df)
+        today = pd.Timestamp.now(tz=self.iso.default_timezone).date()
+        assert df["Interval Start"].max().date() == today
+
+    def test_get_load_today(self):
+        df = self.iso.get_load("today")
+        self._check_load(df)
+        today = pd.Timestamp.now(tz=self.iso.default_timezone).date()
+
+        # okay as long as one of these columns is only today
+        assert (
+            (df["Interval Start"].dt.date == today).all()
+            or (df["Interval Start"].dt.date == today).all()
+            or (df["Interval End"].dt.date == today).all()
+        )
+        return df
+
+    """get_load_forecast"""
+
+    def test_get_load_forecast_historical(self):
+        test_date = (pd.Timestamp.now() - pd.Timedelta(days=14)).date()
+        forecast = self.iso.get_load_forecast(date=test_date)
+        self._check_forecast(forecast)
+
+    def test_get_load_forecast_historical_with_date_range(self):
+        end = pd.Timestamp.now().normalize() - pd.Timedelta(days=14)
+        start = (end - pd.Timedelta(days=7)).date()
+        forecast = self.iso.get_load_forecast(
+            start,
+            end=end,
+        )
+        self._check_forecast(forecast)
+
+    def test_get_load_forecast_today(self):
+        forecast = self.iso.get_load_forecast("today")
+        self._check_forecast(forecast)
+
+    """get_status"""
+
+    def test_get_status_latest(self):
+        # ensure there is a homepage if gridstatus can retrieve a status
+        assert isinstance(self.iso.status_homepage, str)
+
+    """get_storage"""
+
+    @pytest.mark.skip(reason="Not Applicable")
+    def test_get_storage_historical(self):
+        pass
+
+    @pytest.mark.skip(reason="Not Applicable")
+    def test_get_storage_today(self):
+        pass
+
+    def _check_load(self, df):
+        assert isinstance(df, pd.DataFrame)
+        assert df.shape[0] >= 0
+
+        time_type = "interval"
+        self._check_time_columns(df, instant_or_interval=time_type)
+
+        for col in ["Market Total Load", "Ontario Load"]:
+            assert col in df.columns
+            assert is_numeric_dtype(df[col])
+
+    def _check_time_columns(self, df, instant_or_interval="instant"):
```

Comment:
did you have to redefine this because we dont have the `Time` column? if so, let's just remove checking for the `Time` column in the base version. we will eventually remove that from all methods  that return an interval. just made an issue to track that here: https://github.com/kmax12/gridstatus/issues/308

Reply:
Could changing `base_test_iso` wait for a separate PR addressing that linked issue? I'd prefer not to expand the scope too much.

Reply:
Yes, that’s fine by me